plugin/web/extensions/sidebar/core/components/mobile/hooks/index.ts

```diff
@@ -17,7 +17,7 @@ export default () => {
   const [reearthURL, setReearthURL] = useState<string>();
   const [backendURL, setBackendURL] = useState<string>();
   const [backendProjectName, setBackendProjectName] = useState<string>();
-  const [buildingSearch, setBuildingSearch] = useState<BuildingSearch>([]);
+  const [buildingSearch] = useState<BuildingSearch>([]);
```

Comment:
What is going on here? Without the setter function this will forever just be an empty array...

Reply:
The `buildingSearch` state is used for pass into dataset card, from this PR it will not be used when on mobile. I left it here to avoid changing too much to components.
But it could be misleading. I'm removing them now.